From 2afcea70430e65fda214223d1366f15f7d613c98 Mon Sep 17 00:00:00 2001
From: Lukas Schmelting <lschmelting@posteo.com>
Date: Sat, 31 May 2025 16:57:43 +0200
Subject: [PATCH] Disable gs-self-test-flatpak

Currently fails because of unsufficiently setup dbus in the test
environment
---
 plugins/flatpak/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/flatpak/meson.build b/plugins/flatpak/meson.build
index 5f69dfa..f58896a 100644
--- a/plugins/flatpak/meson.build
+++ b/plugins/flatpak/meson.build
@@ -54 +54 @@ if get_option('tests')
-  test('gs-self-test-flatpak', e, suite: ['plugins', 'flatpak'], env: test_env, timeout : 120)
+  # test('gs-self-test-flatpak', e, suite: ['plugins', 'flatpak'], env: test_env, timeout : 120)
-- 
2.49.0

